datanode/sqlstore/migrations/0060_game_details_api.sql

```diff
@@ -0,0 +1,150 @@
+-- +goose Up
+
+alter table transfers add column if not exists game_id bytea null;
+alter table rewards add column if not exists game_id bytea null;
+
+create table game_reward_totals (
+    game_id bytea not null,
+    party_id bytea not null,
+    asset_id bytea not null,
+    market_id bytea not null,
+    epoch_id bigint not null,
+    team_id bytea not null,         -- participant may take part in a game as a team member or as an individual depending on the entity scope of the game
+    total_rewards hugeint not null,
+    primary key (game_id, party_id, asset_id, market_id, epoch_id, team_id)
+);
+
+-- +goose StatementBegin
+create or replace function insert_game_reward_totals()
+returns trigger
+    language plpgsql
+    as $$
+    declare party_team_id bytea;
+begin
+    select team_id into party_team_id from team_members where party_id = new.party_id;
```

Comment:
You have to match the epoch between the membership and the reward, otherwise the stats might be attributed to another team, in case of a team switch.